controllers/powerprofile_controller_test.go

```diff
@@ -877,9 +938,7 @@ func TestPowerProfile_Reconcile_MaxValueLowerThanMinValue(t *testing.T) {
 		}
 
 		_, err = r.Reconcile(context.TODO(), req)
-		if err != nil {
-			t.Errorf("%s failed: expected the reconciler to not have failed", tc.testCase)
-		}
+		assert.ErrorContains(t, err, "max frequency value cannot be lower than the minimum frequency value")
```

Comment:
```suggestion
		assert.ErrorContains(t, err, "max frequency value cannot be lower than the min frequency value")
```

Reply:
Will address all changes to controllers/controller tests in the upcoming release